packages/frontapp/src/app/components/CategorySearch/index.tsx

```diff
@@ -105,11 +53,46 @@ const CategorySearch: FC<IProps> = ({ setActive }) => {
 
   useEffect(() => {
     if (allAuthors && allCategories && allTypes && allAvailability) {
-      FilteringObjects();
+      setAttributes([
```

Comment:
I used to do the same way, but was told that its a little bit too messy, so its better to keep them in separate function

Reply:
I could move the array out so it will look prettier